content/en/agent/guide/private-link.md

```diff
@@ -27,37 +27,37 @@ The overall process consists of configuring an internal endpoint in your VPC tha
 1. Connect to the AWS console and create a new VPC endpoint:
    {{< img src="agent/guide/private_link/create_vpc_endpoint.png" alt="Create VPC endpoint" style="width:60%;" >}}
 2. Select **Find service by name**.
-3. Fill the _service name_ text box with the following value depending of which private link you want to setup:
+3. Fill the _service name_ text box with the following value, depending on which private link you want to set up:
```

Comment:
nit: service name -> Service Name

![image](https://user-images.githubusercontent.com/5319942/75369108-687c6200-5888-11ea-8bc8-8c4f2f7ef9c3.png)

Possible alternate wording: "depending on which service you want to establish AWS PrivateLink for" ?  Not sure if that's more or less awkward, but am unsure about using "private link" like this.

Reply:
Good points, editing